Source/FolioReaderFontsMenu.swift

```diff
@@ -243,7 +243,12 @@ class FolioReaderFontsMenu: UIViewController, SMSegmentViewDelegate, UIGestureRe
             scrollDirection = readerConfig.scrollDirection
         }
 
-        layoutDirection.selectSegmentAtIndex(scrollDirection?.rawValue ?? 0)
```

Comment:
It won't crash if you have different defaults already set for configuration. See this line:

https://github.com/FolioReader/FolioReaderKit/pull/237/files#diff-7d83fdc08c25740f0dc3bab59bd7c2bbL72

It sets the default value to `defaultVertical`. In this line it takes the default value if nothing has changed it:

https://github.com/FolioReader/FolioReaderKit/pull/237/files#diff-21569209c801652e20649d5072f824d0R255

And then the conditional in this line does not get triggered:

https://github.com/FolioReader/FolioReaderKit/pull/237/files#diff-21569209c801652e20649d5072f824d0R255

Which means that when you reach this line:

https://github.com/FolioReader/FolioReaderKit/pull/237/files#diff-21569209c801652e20649d5072f824d0R255

`scrollDirection` is still `defaultVertical`, which has a `rawValue` of 3. There's only two segments so it crashes.

I expect you have some unique state from developing the project that set your current scroll direction to either `horizontal` or `vertical`. I think user defaults are maintained across app deletion, so to reproduce you would have to either
 - manually clear out user defaults programmatically, then relaunch to let the defaults get set again, or
 - reset a simulator to start fresh on it